manifests/params.pp

```diff
@@ -41,6 +41,9 @@
   $fork_remote_name = undef
   $upstream_remote_name = 'upstream'
 
+  $foreman_scm_revision = 'develop'
+  $katello_scm_revision = 'master'
```

Comment:
I wonder if there's a list of places we need to change when this changes to `main`..

Reply:
Nonexistent afaik